PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/directives/tabs.txt

- Eliminate the need for the writer to specify the tab display name
  (tab label)

> Refer to the `Snooty configuration file <https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml>`__
>  to find the pre-defined tab sets. Each of the entries starts with
> ``[directive.tabs-``.

Reviewer: ```suggestion
To find the pre-defined tab sets, see the `Snooty configuration file
<https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml>`__.
Each of the entries starts with ``[directive.tabs-``.
```

Defining a New Tab Set
~~~~~~~~~~~~~~~~~~~~~~

> Reach out to the {+docs-platform+} for more information on creating a new
> pre-defined tab set. 

Reviewer: ```suggestion
To learn more about creating a new pre-defined tab set, 
contact the {+docs-platform+}. 
```
